tartufo/types.py

```diff
@@ -103,6 +103,10 @@ class ScanException(TartufoException):
     """Raised if there is a problem encountered during a scan"""
 
 
+class BranchNotException(TartufoException):
+    """Raised if a branch was not found"""
```

Comment:
Per above, I'd make this `GitBranchException(GitException)` and you might need to move it down if this is ahead of the `GitException` declaration.